File: irshad dir/kamal/demo_python/python/L6/circle.py
# #pp2  class circle
# data  radius
# method area(),circum() and default initilizer()
# radius from user and print area and circum 

import math
class Circle:
	def __init__(self):
		self.radius=0.0
	def area(self):
		ans=math.pi*math.pow(self.radius,2)
		print("area= ",ans)
	def circum(self):
		ans=2*math.pi*self.radius
		print("circumference = ",ans)
		
rad=float(input("enter radius "))
c=Circle()
c.radius=rad
c.area()
c.circum()

File: irshad dir/kamal/demo_java/java/L7/Circle.java
import java.io.*;
class Circle
{
public static void main(String args[])
{
Console c=System.console();
double r=Double.parseDouble(c.readLine("Enter Radius:"));
double area=Math.PI*Math.pow(r,2);
System.out.printf("area %.3f%n",area);
System.out.println("area "+area);
}
}